Sources/Deboogger.swift

```diff
@@ -47,34 +47,61 @@ public final class Deboogger {
         return window
     }()
 
-    private weak var pluginViewController: PluginViewController?
+    weak var pluginViewController: PluginViewController?
     public var viewController: UIViewController? {
         return pluginViewController
     }
     public var navigationController: UINavigationController? {
         return pluginViewController?.navigationController
     }
 
+    private var configuration: Configuration?
+    private var isCurrentlyShowing: Bool = false
+
+    private var _shouldShowAssistiveButton: Bool = false
+    var shouldShowAssistiveButton: Bool {
+        get {
+            #if targetEnvironment(simulator)
+                return true
+            #endif
+            return _shouldShowAssistiveButton
+        }
+        set {
+            _shouldShowAssistiveButton = newValue
+        }
+    }
+
     private init() {}
 
     // MARK: - Configurations
 
     public static func configure(with plugins: [Plugin]) {
-        shared.configure(with: PluginsConfiguration(plugins: plugins))
+        #if targetEnvironment(simulator)
+            shared.configure(with: PluginsConfiguration(plugins: plugins))
+        #else
+            let section = Section(title: "App plugins", plugins: plugins)
+            configure(with: [section])
```

Comment:
```suggestion
            configure(with: section)
```
I guess it should work as well

Reply:
Yes, it should. But isn't it better to call function already instead of chaining them?